### PR TITLE
Add preview on time table

### PR DESCRIPTION
**Pull request by hiesiea**
## Issue
- none

## Overview (Required)
- Add preview on time table

## Links
- none

## Screenshot (Optional if no screenshot test or unrelated to UI)
BookmarkTopArea | EmptySearchResultBody
:--: | :--:
BookmarkTopArea | EmptySearchResultBody 

SearchFilter | TimetableItemDetailBottomAppBar
:--: | :--:
SearchFilter | TimetableItemDetailBottomAppBar

TimetableItemDetailScreenTopAppBar | TimetableItemDetailSummaryCardRow | TimetableTab | TimetableTopArea
:--: | :--: | :--: | :--:
TimetableItemDetailScreenTopAppBar | TimetableItemDetailSummaryCardRow | TimetableTab | TimetableTopArea

### CODE REVIEW

**Review by momomomo111 — APPROVED** [review]
Thanks for providing screenshots as well!
The changes are very easy to understand and very nice!

File: core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2023/model/DroidKaigi2023Day.kt [hunk 1]
```diff
@@ -7,11 +7,11 @@ import kotlinx.datetime.TimeZone
 import kotlinx.datetime.toInstant
 import kotlinx.datetime.toLocalDateTime
 
-public enum class DroidKaigi2023Day(
-    public val day: Int,
-    public val dayOfMonth: Int,
-    public val start: Instant,
-    public val end: Instant,
+enum class DroidKaigi2023Day(
```
Comment (takahirom) [inline core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2023/model/DroidKaigi2023Day.kt:10]: Maybe we are going to use explicit mode. But we can fix it later 🙏

Comment (takahirom) [reply core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2023/model/DroidKaigi2023Day.kt:10]: Sorry, on second thoughts. Could you revert this change..? 🙇

Comment (hiesiea) [reply core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2023/model/DroidKaigi2023Day.kt]: Sorry too. I fixed it.
[3240b94](https://github.com/DroidKaigi/conference-app-2023/pull/818/commits/3240b9428161e75c0bf5d2cd5ee4aa089d8a5ba7)

**Comment by hiesiea** [comment]
> please run ./gradlew detekt --auto-correct

I ran this command. However, the code was not modified.
What should I do?

```
./gradlew detekt --auto-correct
Configuration on demand is an incubating feature.
Type-safe project accessors is an incubating feature.

> Configure project :
Project accessors enabled, but root project name not explicitly set for 'build-logic'. Checking out the project in different folders will impact the generated code and implicitly the buildscript classpath, breaking caching.

> Task :build-logic:compileKotlin
w: file:///Users/hitoshi_kamada/StudioProjects/conference-app-2023/build-logic/src/main/kotlin/io/github/droidkaigi/confsched2023/primitive/AndroidGradleDsl.kt:80:9 'setter for config: ConfigurableFileCollection' is deprecated. Setter will be removed in a future release. Use `from` or `setFrom` instead.
w: file:///Users/hitoshi_kamada/StudioProjects/conference-app-2023/build-logic/src/main/kotlin/io/github/droidkaigi/confsched2023/primitive/AndroidGradleDsl.kt:96:48 Unchecked cast: TaskProvider<Task!> to TaskProvider<ReportMergeTask>

> Configure project :core:model
w: file:///Users/hitoshi_kamada/StudioProjects/conference-app-2023/core/model/build.gradle.kts:22:13: Variable 'androidMain' is never used

> Task :core:ui:detekt
Property 'style>MandatoryBracesIfStatements' is deprecated. Use `BracesOnIfStatements` with `always` configuration instead.

> Task :core:common:detekt
Property 'style>MandatoryBracesIfStatements' is deprecated. Use `BracesOnIfStatements` with `always` configuration instead.

> Task :core:designsystem:detekt
Property 'style>MandatoryBracesIfStatements' is deprecated. Use `BracesOnIfStatements` with `always` configuration instead.

> Task :feature:about:detekt
Property 'style>MandatoryBracesIfStatements' is deprecated. Use `BracesOnIfStatements` with `always` configuration instead.

> Task :core:testing:detekt
Property 'style>MandatoryBracesIfStatements' is deprecated. Use `BracesOnIfStatements` with `always` configuration instead.

> Task :app-android:detekt
Property 'style>MandatoryBracesIfStatements' is deprecated. Use `BracesOnIfStatements` with `always` configuration instead.

> Task :core:model:detekt
Property 'style>MandatoryBracesIfStatements' is deprecated. Use `BracesOnIfStatements` with `always` configuration instead.

> Task :core:data:detekt
Property 'style>MandatoryBracesIfStatements' is deprecated. Use `BracesOnIfStatements` with `always` configuration instead.

> Task :feature:floor-map:detekt
Property 'style>MandatoryBracesIfStatements' is deprecated. Use `BracesOnIfStatements` with `always` configuration instead.

> Task :feature:main:detekt
Property 'style>MandatoryBracesIfStatements' is deprecated. Use `BracesOnIfStatements` with `always` configuration instead.

> Task :feature:sponsors:detekt
Property 'style>MandatoryBracesIfStatements' is deprecated. Use `BracesOnIfStatements` with `always` configuration instead.

> Task :feature:sessions:detekt
Property 'style>MandatoryBracesIfStatements' is deprecated. Use `BracesOnIfStatements` with `always` configuration instead.

> Task :feature:staff:detekt
Property 'style>MandatoryBracesIfStatements' is deprecated. Use `BracesOnIfStatements` with `always` configuration instead.

> Task :feature:stamps:detekt
Property 'style>MandatoryBracesIfStatements' is deprecated. Use `BracesOnIfStatements` with `always` configuration instead.

> Task :reportMerge
Merged XML output to /Users/hitoshi_kamada/StudioProjects/conference-app-2023/build/reports/detekt/merge.xml

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.2.1/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 53s
20 actionable tasks: 18 executed, 2 up-to-date

~/StudioProjects/conference-app-2023 feature/add_preview_on_time_table 54s
❯ 

~/StudioProjects/conference-app-2023 feature/add_preview_on_time_table
❯ git status                                       
On branch feature/add_preview_on_time_table
nothing to commit, working tree clean

```

**Comment by takahirom** [comment]
Sorry we need to use spotlessKotlinApply
We have fixed it
https://github.com/DroidKaigi/conference-app-2023/pull/863

**Comment by takahirom** [comment]
I misunderstood.
We might have to migrate to detekt

**Comment by hiesiea** [comment]
@takahirom 
Thanks! I fixed it.

**Review by takahirom — APPROVED** [review]
Looks great! We'll be able to develop easily!

**Pull request merged**